app/controllers/api/V1/doctors_controller.rb

```diff
@@ -0,0 +1,52 @@
+class Api::V1::DoctorsController < ApplicationController
+  before_action :set_doctor, only: %i[show destroy]
+
+  def index
+    render json: Doctor.all
+  end
+
+  def show
+    if @doctor.blank?
+      render json: { error: "Doctor with id #{params[:id]} does not exist" }, status: 404
+    else
+      render json: @doctor
+    end
+  end
+
+  def create
+    doctor = Doctor.new(doctor_params)
+    doctor.age = age
+
+    if doctor.save
+      render json: doctor, status: 201
+    else
+      render json: { error: doctor.errors }, status: 422
+    end
+  end
+
+  def destroy
+    if @doctor.blank?
+      render json: { error: "Doctor with id #{params[:id]} not found" }, status: 404
+    else
+      @doctor.delete
+      render status: 204
+    end
+  end
```

Comment:
With the delete action, we have a bog as well, in this case, is about cors permission, so I believe it just needs to give it the right permissions to allow the action, here is the API response:

![Captura de Pantalla 2022-03-28 a la(s) 9 07 15](https://user-images.githubusercontent.com/88604987/160431022-62cda079-4bf6-4429-804d-fc72953a95ac.png)

Kindly give it the right cors permissions 🤓  ✅

Reply:
Thank you. I'll fix this CORS issue immediately.